src/bindgen/ir/opaque.rs

```diff
@@ -20,33 +20,56 @@ use bindgen::writer::{Source, SourceWriter};
 
 #[derive(Debug, Clone)]
 pub struct OpaqueItem {
-    pub name: Path,
+    pub path: Path,
+    pub export_name: String,
     pub generic_params: GenericParams,
     pub cfg: Option<Cfg>,
     pub annotations: AnnotationSet,
     pub documentation: Documentation,
 }
 
 impl OpaqueItem {
-    pub fn new(
-        name: String,
+    pub fn load(
```

Comment:
Is there a reason this function is needed? Seeing as it just returns `Ok(OpaqueItem::new(..))`

Reply:
It's merely to make `OpaqueItem` be more in line with the rest of `ir::`.

And to hide the …

```rust
…
LiteralExpr::load(&item.expr)?,
Cfg::append(mod_cfg, Cfg::load(&item.attrs)),
AnnotationSet::load(&item.attrs)?,
Documentation::load(&item.attrs),
…
```

… shenanigans. Which is what all others are doing in `fn load(…)`, too.